jdk/make/CopyFiles.gmk

```diff
@@ -323,6 +323,29 @@ ifneq ($(OPENSSL_BUNDLE_LIB_PATH), )
 
     COPY_FILES += $(OPENSSL_TARGET_LIB)
   endif
+
+  ifeq (true,$(OPENJ9_ENABLE_JITSERVER))
+    # Copy OpenSSL SSL Lbrary
+    # To bundle first search for libssl 1.1.x library, if not found, search for 1.0.x
+    ifeq ($(OPENJDK_TARGET_OS), linux)
+      OPENSSL_SSL_LIB_NAME = libssl.so.1.1
+      ifeq ("$(wildcard $(OPENSSL_BUNDLE_LIB_PATH)/$(OPENSSL_SSL_LIB_NAME))", "")
```

Comment:
If openssl is not bundled, `OPENSSL_BUNDLE_LIB_PATH` will not be set: how will this work?

Reply:
In the case where openssl is not bundled and`OPENSSL_BUNDLE_LIB_PATH` is not set: 

1. If `--with-openssl=fetched` is specified, `OPENSSL_DIR ` is set. JITServer is built against the fetched OpenSSL under `OPENSSL_DIR`. This is the main use case since the PR Jenkins build specifies `--with-openssl=fetched`. 

2. If `--with-openssl=system` is specified or `--with-openssl` is not set at all, JITServer is built against the installed OpenSSL on the build system. Without this change, this is what we have now by default.

Reply:
It should be possible to add jitserver SSL support independent of whether the extra libraries are bundled or not, and independent of where the required libraries are on the build system.

Reply:
Similar reply as https://github.com/eclipse/openj9/pull/7846#discussion_r351011230